docs/content/en/docs/Customizing/configure_grub.md

```diff
@@ -28,10 +28,10 @@ menuentry "cOS" --id cos {
 ```
 
 {{% alert title="Kernel parameters" %}}
-The kernel parameters are not part of the persistent `grub.cfg` file stored in `COS_RECOVERY` partition. 
-Kernel parameters are sourced from the loop device of the OS image to boot. This is mainly to
-keep kernel parameters consistent across different potential OS images or
-system upgrades. 
+The kernel parameters are not part of the persistent `grub.cfg` file stored in
+`COS_RECOVERY` partition. Kernel parameters are sourced from the loop device of
```

Comment:
The parameters are now always in COS_STATE, this was already wrong before...

Reply:
Oh I missed that! Nice catch!